pkg/commands/process/settings/rules.go

```diff
@@ -240,3 +275,21 @@ func buildRules(definitions map[string]RuleDefinition, enabledRules map[string]s
 
 	return rules
 }
+
+func cachedRulesExist(bearerRulesDir string) bool {
+	_, err := os.Stat(bearerRulesDir)
+	return err == nil
+}
+
+func cleanupRuleDirFiles(bearerRulesDir string) error {
+	os.RemoveAll(bearerRulesDir)
+	return nil
+}
+
+func bearerRulesDir() string {
+	tmpDir := os.TempDir()
+	if tmpDir == "/tmp" {
```

Comment:
Why is the special case needed?

Reply:
Ah, thanks - it's not 🙈 It was from me trying to get the CI to be happy, but we can remove it 👍